.gitlab-ci.yml

```diff
@@ -495,6 +495,72 @@ TCID-LOCALPV-HOSTPATH-CREATE:
     - chmod 755 ./stages/functional/local-pv-hostpath/local-pv-hostpath
     - ./stages/functional/local-pv-hostpath/local-pv-hostpath
 
+TCID-ZFS-LOCALPV-PROVISIONER-DEPLOY:
+  extends: .func_test_template
+  script:
+    - chmod 755 ./stages/functional/ZFS-LocalPV-provisioner/ZFS-LocalPV-provisioner
+    - ./stages/functional/ZFS-LocalPV-provisioner/ZFS-LocalPV-provisioner
```

Comment:
Rename the directory to `TCID-ZFS-LOCALPV-PROVISIONER-DEPLOY`. This will make it easier to identify the test script in the repository.

Reply:
Done !